### DISCUSSION TITLE
Optimizer using the wrong gpu

### DISCUSSION THREAD

**Discussion by OGEpsilon** [MaxLastBreath/nx-optimizer#167] [Q&A] [answered]
So I have two gpus in my pc an RX 7800 XT and a GTX 1650 and the optimizer defaults to the 1650 even though i want to run it on my 7800. is there a way to change that or add a feature to select gpu prioritization?? I've tried setting Ryujinx as an app to run off of my 7800 but because it launches using the optimizer i think that that overrides it. Please provide answers for my problem, my head is spinning from trying to figure this out.

**Comment by MaxLastBreath** [comment, marked as answer]
hey the optimizer does absolutely nothing with your GPU, it literally cannot override what the emulator is using, if you are having issues with the emulator choosing the wrong GPU, you way want to contact the Ryujinx team. All the optimizer does is just reports what it is able to detect, for the sake of debugging stuff with the mod itself.
